lib/index.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
-const zstd = require('bindings')('zstd');
+const { resolve } = require('path');
 const { promisify } = require('util');
 const { isUint8Array } = require('util/types');
 
+const zstd = require(resolve(__dirname, '../build/Release/zstd.node'));
```

Comment:
```suggestion
const zstd = require('../build/Release/zstd.node');
```

No need to run the same resolution logic that require already has, plus the string literal is desirable to fix issues with bundlers (you can ignore/customize a hard-coded string easily)